t/data/tests/main.pm

```diff
@@ -34,7 +34,7 @@ if (!$integration_tests && eval { require Inline::Python }) {
     autotest::loadtest "tests/pre_boot.py";
 }
 
-autotest::loadtest "tests/boot.pm";
```

Comment:
The Lua version should probably be loaded only if Lua support is enabled (when `Inline::Lua` is available).